R/process_data_set_sunni.R

```diff
@@ -0,0 +1,36 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+library(snakecase)
+
+## wwc_matches ---------------------------------------------------------------------------------------------------------
+wwc_2019_matches <- readr::read_csv("https://projects.fivethirtyeight.com/soccer-api/international/2019/wwc_matches.csv") %>%
+  clean_names() %>%
+  select(-c(league, league_id)) %>%
+  mutate(
+    team1 = to_snake_case(team1),
+    team2 = to_snake_case(team2)
+    )
+
+usethis::use_data(wwc_2019_matches, overwrite = TRUE)
+
+## wwc_forecasts ---------------------------------------------------------------------------------------------------------
+wwc_2019_forecasts <- readr::read_csv("https://projects.fivethirtyeight.com/soccer-api/international/2019/wwc_forecasts.csv") %>%
+  clean_names() %>%
+  mutate(
+    team = to_snake_case(team),
+    date = as.Date(forecast_timestamp),
+    group = as.factor(group),
+    team = as.factor(team)
+    ) %>%
+  select(-c(forecast_timestamp, timestamp))
+
+wwc_2019_forecasts <- wwc_2019_forecasts[c(
```

Comment:
@sunniraleigh What is this doing? Reordering the variables? Is there a reason you are using `[` instead of `select()` (as above)?

Reply:
I wanted to reorder the variables so that my new mutated variable was in the front.

Reply:
OK. Perhaps @rudeboybert can weigh in on whether he wants the variable order to match what FiveThirtyEight has.

Reply:
Hi @sunniraleigh, please try to keep the same order as the original 538 data if possible